app/Models/User.php

```diff
@@ -66,10 +66,10 @@ function groups_owned()
 
     public function getFullNameAttribute()
     {
-        $name_parts = array_filter([$this->first_name, $this->last_name]);
-        if ($name_parts)
+        $nameParts = array_filter([$this->first_name, $this->last_name]);
```

Comment:
Why are you filtering these??.... Are you not requiring both at the User creation level??... 🤔 I can't figure out why you would do this...

Reply:
They are both required. I guess it's a bit extra here.

Are you saying, it can just be `$this->first_name.' '.$this->last_name`, and keep it simple?

Reply:
Yeah. The array filter is only needed to filter out `null` value'd keys.

```
array_filter([null, 42]) // [42]
```

Just concatenate & return imho